notes.md

```diff
@@ -0,0 +1,127 @@
+# aic-kube MaaS Orchestration #
```

Comment:
Could either clean this up and incorporate it into the README, open an "Design Discussion" issue and place this there, or keep it locally